package.json

```diff
@@ -66,7 +66,7 @@
     "tempy": "^0.2.1",
     "tildify": "^1.2.0",
     "tmp": "^0.0.33",
-    "unstated": "^1.2.0"
+    "unstated": "^2.1.1"
```

Comment:
Hmmm, I didn't realize that. I upgraded to stop using `this.state` in `this.setState()` since passing the function wasn't available in the version we were using. Since this will require some cleanup, I think I'll bring it back down, do it the "bad" way, add a TODO comment like the rest and then open a new PR/Issue to upgrade unstated and make sure it works everywhere

Reply:
This is most likely what's causing the error in the cropper as well

Reply:
Yup